src/renderer/qssWidget/js/qssMySavedSettingsWidget.js

```diff
@@ -61,7 +61,7 @@
                     model: {
                         label: "{mySavedSettings}.model.messages.reApplyPreferencesButtonLabel"
                     },
-                    invokers: {
+                    listeners: {
                         "onClick": "{channelNotifier}.events.onQssReApplyPreferencesRequired.fire"
```

Comment:
This listener should have a namespace otherwise it can't be overridden. Something like "onClick.reApplyPreferences"

Reply:
indeed, done